src/Components/PatientPage/Header/Header.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+
+
+export const Header = (props) => {
```

Comment:
why do you have two Header components? don`t it the same?

Reply:
It will be refactor after creating StyledHeader